sormas-backend/src/main/java/de/symeda/sormas/backend/user/UserFacadeEjb.java

```diff
@@ -953,6 +956,9 @@ private List<User> getUsersFromCasesByDistricts(List<String> districtsUuidsAmong
 				Arrays.asList(UserRight.CASE_RESPONSIBLE))
 			.stream()
 			.map(userReference -> userService.getByUuid(userReference.getUuid()))
+			.collect(Collectors.toList())
+			.stream()
```

Comment:
no need to collect and stream again, you can just filter after the mapping, it's more performant because it does less iterations over the list

Reply:
removed one stream